### PR TITLE
DOC: Example of data output for archival 

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Addresses #6

Includes basic examples in the docs and links to the ops_reach project for people who want to build archivable data products.

# Type of change

- This change requires a documentation update

# How Has This Been Tested?

Inspection at the docs: https://pysatnasa.readthedocs.io/en/doc-6_example/archival.html

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes
- [x] Update zenodo.json file for new code contributors

If this is a release PR, replace the first item of the above checklist with the release
checklist on the wiki: https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: docs/archival.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  pysat.utils.io.inst_to_netcdf(reach, 'output_file.nc', epoch_name='Epoch',
+                                meta_translation=meta_dict,
+                                export_pysat_info=False)
+
+
+In this case, note that the pysat 'name' label is output to three different
+metadata values required by the ITSP standards. Additionally, the
+``export_pysat_info`` option is set to false here. This drops several internal
+pysat metadata values before writing to file.
+
```
Comment (jklenzing) [inline docs/archival.rst:55]:
```suggestion

A full guide to SPDF metadata standards can be found `here <https://spdf.gsfc.nasa.gov/istp_guide/istp_guide.html>`_.
```

File: docs/archival.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+A full guide to SPDF metadata standards can be found `here <https://spdf.gsfc.nasa.gov/istp_guide/istp_guide.html>`_.
```
Comment (jklenzing) [inline docs/archival.rst:56]:
```suggestion
A full guide to SPDF metadata standards can be found 
`here <https://spdf.gsfc.nasa.gov/istp_guide/istp_guide.html>`_.

```

**Comment by jklenzing** [comment]
@t-esman, if you've run into other things while working on ops_maven, this would be a good place to document.

**Review by t-esman — APPROVED** [review]

File: docs/archival.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+The codes and routines at :py:mod:`pysatNASA` are designed for end-users of NASA data
+products. However, pysat in general has also been used to build operational
+instruments for generating archival data to be uploaded to the Space Physics
+Data Facility (SPDF) at NASA.
+
+In general, such instruments should include separate naming conventions. An
+example of this is the REACH data, where netCDF4 files are generated for
+archival purposes as part of the `ops_reach` package, but can be accessed by
+the end user through pysatNASA.
```
Comment (jklenzing) [inline docs/archival.rst:12]:
```suggestion
the end user through :py:mod:`pysatNASA`.
```

File: docs/archival.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+=============================================
+
+The codes and routines at :py:mod:`pysatNASA` are designed for end-users of NASA data
+products. However, pysat in general has also been used to build operational
+instruments for generating archival data to be uploaded to the Space Physics
+Data Facility (SPDF) at NASA.
+
+In general, such instruments should include separate naming conventions. An
+example of this is the REACH data, where netCDF4 files are generated for
+archival purposes as part of the `ops_reach` package, but can be accessed by
```
Comment (jklenzing) [inline docs/archival.rst:11]:
```suggestion
archival purposes as part of the :py:mod:`ops_reach` package, but can be accessed by
```

File: docs/archival.rst [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+the end user through pysatNASA.
+
+In general, a ``pysat.Instrument`` object can be constructed for any dataset.
+Full instructions and conventions can be found
```
Comment (jklenzing) [inline docs/archival.rst:15]:
```suggestion
In general, a :py:class:`pysat.Instrument` object can be constructed for any
dataset. Full instructions and conventions can be found
```

File: docs/archival.rst [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+  pysat.utils.io.inst_to_netcdf(reach, 'output_file.nc', epoch_name='Epoch',
+                                meta_translation=meta_dict,
+                                export_pysat_info=False)
+
+
+In this case, note that the pysat 'name' label is output to three different
+metadata values required by the ITSP standards. Additionally, the
+``export_pysat_info`` option is set to false here. This drops several internal
+pysat metadata values before writing to file.
```
Comment (jklenzing) [inline docs/archival.rst:54]:
```suggestion
:py:attr:`export_pysat_info` option is set to false here. This drops several
internal :py:mod:`pysat` metadata values before writing to file.
```

**Review by aburrell — APPROVED** [review]

**Pull request merged**